### PR TITLE
Adding featured event logic for city search

### PR DESCRIPTION
**Pull request by unbeatablesites**
Hope you like!

### CODE REVIEW

File: assets/js/main.js [hunk 1]
```diff
@@ -1,172 +1,69 @@
...
-			small:   [ '481px',  '736px'  ],
-			xsmall:  [ null,     '480px'  ]
-		});
 
-	// Play initial animations on page load.
-		$window.on('load', function() {
-			window.setTimeout(function() {
-				$body.removeClass('is-preload');
-			}, 100);
-		});
```
Comment (annievallone) [inline assets/js/main.js:22]: Did you move this logic somewhere else or was it no longer needed? In general when making these types of changes you may want to add comments for why you are adding or removing logic to reduce back and forth

File: assets/js/main.js [hunk 2]
```diff
@@ -1,172 +1,69 @@
...
-						$wrapper
-							.css('margin-top', -25)
-							.css('padding-bottom', 25);
+for (var i = 0; i < response._embedded.events.length; i++) {
 
-				// Pass scroll event to window.
-					$wrapper.on('scroll', function() {
-						$window.trigger('scroll');
-					});
+            // var newB = $("<button>");
```
Comment (annievallone) [inline assets/js/main.js:17]: If this var is not needed pls remove the line instead of commenting it out

File: assets/js/main.js [hunk 3]
```diff
@@ -1,172 +1,69 @@
...
-				$window.on('load.hl_scrolly', function() {
 
-					$('.scrolly').scrolly({
-						speed: 1500,
-						parent: $wrapper,
-						pollOnce: true
-					});
 
-					$window.off('load.hl_scrolly');
+            $("#movies").append(replace);
```
Comment (annievallone) [inline assets/js/main.js:27]: Is this ID "movies" intended to show the event? Why did you choose movie instead of event?

**Review by annievallone — CHANGES_REQUESTED** [review]
I added a few comments. If you can review and respond pls

File: assets/js/main.js (hunk 1 above)
Comment (unbeatablesites) [reply assets/js/main.js:22]: It's no longer needed. I'm making sure the site s clean and loads fast!

File: assets/js/main.js (hunk 2 above)
Comment (unbeatablesites) [reply assets/js/main.js:17]: Ok, I'll remove my commented out lines.

File: assets/js/main.js (hunk 3 above)
Comment (unbeatablesites) [reply assets/js/main.js:27]: It's just a place holder I'll change it to events.

**Review by unbeatablesites — COMMENTED** [review]
I'll make the changes when I get to my lap top this evening :)

**Comment by unbeatablesites** [comment]
I've made many commits since posting this version. Many things have changed. The sooner we get the updates the better because I've added many features that others my need to see.

**Review by annievallone — APPROVED** [review]

**Review by Surrealinsomniac — APPROVED** [review]

**Pull request merged**